### PR TITLE
Feat: add custom title bar

### PR DESCRIPTION
**Pull request by p0lygun**
Added custom title bar and widget root

image

### CODE REVIEW

**Comment by p0lygun** [comment]
@noahlias review pls 😀

File: client/app/lib/kivy_manager.py [hunk 1]
```diff
@@ -1,23 +1,80 @@
...
+
+class ClientUI(MDApp):
     """Main Class to Build frontend on."""
 
     ws: websockets.WebSocketClientProtocol = None
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.ws_handler_task = None
+        self.title = "Blak"
```
Comment (noahlias) [inline client/app/lib/kivy_manager.py]: build method has init the variable title, unnecessary to init it

Comment (p0lygun) [reply client/app/lib/kivy_manager.py]: this is more verbose but ok

Comment (noahlias) [reply client/app/lib/kivy_manager.py]: ok

**Review by noahlias — DISMISSED** [review]

**Review by noahlias — APPROVED** [review]

**Pull request merged**